pg-promise-exercises/exercises.js

```diff
@@ -0,0 +1,224 @@
+const pg = require('pg-promise')()
+const assert = require('assert')
+
+const postgresConfig = {
+  host: 'localhost',
+  port: 5432,
+  database: 'pg-promise-exercises',
+  user: '<change-this-to-your-username>', // replace this with your username
+  password: '' //  replace this if you have set a password for your username (this is unlikely)
+};
+
+
+const db = pg(postgresConfig);
+
+/* -----------------------------------------
+   Exercise 1
+   -----------------------------------------
+
+   This is an example function that finds all the books from the `books` table
+   @function: `allBooks`
+   @input params: None
+   @output: [{id, title, author_id, subject_id}]
+
+   The assertion fails, and you have to make it pass.
+
+*/
+
+
+
+const allBooks = db.any('select * from books')
```

Comment:
db.any is risky if you're expecting a certain outcome. If you are planning on getting an array back then db.many would be less ambiguous.